core/src/main/kotlin/pages/RendererSpecificPage.kt

```diff
@@ -1,8 +1,12 @@
 package org.jetbrains.dokka.pages
 
+import org.jetbrains.dokka.DokkaConfiguration
+import org.jetbrains.dokka.links.DRI
 import org.jetbrains.dokka.renderers.Renderer
 import kotlin.reflect.KClass
 
+typealias LocationResolver = (DRI, Set<DokkaConfiguration.DokkaSourceSet>) -> String
```

Comment:
Imho this is the most questionable line of this PR. The reason behind it is that `LocationProvider` is introduced on the `base` plugin level and here we are not even in plugin